nvme-rpmb.c

```diff
@@ -629,9 +629,10 @@ static int rpmb_program_auth_key(int fd, unsigned char target,
 	/* reuse response buffer */
 	memset(rsp, 0, rsp_size);
 	err = recv_rpmb_rsp(fd, req->target, rsp_size, rsp);
-	if (err != 0) {
-		err = check_rpmb_response(req, rsp, "Failed to Program Key");
-	}
+	if (err != 0)
+		fprintf(stderr, "Program Key recv error = 0x%x\n", err);
+	else 
+    		err = check_rpmb_response(req, rsp, "Failed to Program Key");
 out:
```

Comment:
Please follow the style from above, test for error print warning bail out. Also check the coding style. The commit message needs also more attention.